assets/js/atomic/blocks/product-elements/rating/block.tsx

```diff
@@ -40,12 +40,35 @@ const getRatingCount = ( product: ProductResponseItem ) => {
 	return Number.isFinite( count ) && count > 0 ? count : 0;
 };
 
+const getStarStyle = ( rating: number ) => ( {
+	width: ( rating / 5 ) * 100 + '%',
+} );
+
+const NoRating = ( { parentClassName }: { parentClassName: string } ) => {
+	const starStyle = getStarStyle( 0 );
+
+	return (
+		<div
+			className={ classnames(
+				'wc-block-components-product-rating__norating-container',
+				`${ parentClassName }__product-rating__norating-container`
```

Comment:
> I'm not sure that I followed the BEM convention correctly. Please, double-check it!

BEM convention allows only one element nesting, so according to it, this structure: `block__element__element` is not recommended (e.g. https://scalablecss.com/bem-nesting-grandchild-elements/).

I don't know the exact purpose of the class in here, but to my best knowledge it should be:

```suggestion
				`${ parentClassName }-product-rating__norating-container`
```

Reply:
Thanks for the suggestion!